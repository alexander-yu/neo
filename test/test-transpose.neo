void main(){
    create matrix<int> a = [[1,2],[3,4],[5,6]];
    print(transpose(a));
    print("\n");
    create matrix<float> b = [[1.5,4.5,4.2,3.1],[1.,-4.2,5.3,6.1]];
    print(transpose(b));
}

