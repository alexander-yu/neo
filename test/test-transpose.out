[[1, 3, 5],
 [2, 4, 6]]
[[1.5, 1],
 [4.5, -4.2],
 [4.2, 5.3],
 [3.1, 6.1]]
